optaplanner-examples/src/main/resources/org/optaplanner/examples/machinereassignment/machineReassignmentSolverConfig.xml

```diff
@@ -27,6 +24,33 @@
     <customPhaseCommandClass>org.optaplanner.examples.machinereassignment.solver.solution.initializer.ToOriginalMachineSolutionInitializer</customPhaseCommandClass>
   </customPhase>
   <localSearch>
+    <termination>
+      <unimprovedSecondsSpentLimit>1</unimprovedSecondsSpentLimit>
+    </termination>
+    <unionMoveSelector>
+      <changeMoveSelector/>
+      <swapMoveSelector/>
+    </unionMoveSelector>
+    <acceptor>
+      <entityTabuSize>7</entityTabuSize>
+      <!--<lateAcceptanceSize>2000</lateAcceptanceSize>-->
+    </acceptor>
+    <forager>
+      <acceptedCountLimit>2000</acceptedCountLimit>
+      <!--<acceptedCountLimit>500</acceptedCountLimit>-->
+    </forager>
+  </localSearch>
+
+  <ruin/>
```

Comment:
Instead of putting the ruin configs in the default solver configs, I'd remove it from here and create a new general benchmark config [here](https://github.com/kiegroup/optaplanner/tree/main/optaplanner-examples/src/main/resources/org/optaplanner/examples/app/benchmark), allowing to compare:

- Multiple examples
- on multiple datasets
- with ruin+nearby enabled, simple ruin enabled, and ruin disabled.

Then you can run the benchmarker once, let it run for hours, and have a comprehensive report.
I'm assuming this report will help you in your thesis just as well.

I also have an additional reason for doing it like this - I don't want to expose the ruin phase by default. I want to give users the opportunity to play with it, before we make it "officially official". So we first introduce it in a less prominent space, and only put it in the primary solver configs a few releases later.

Reply:
the benchmark config is there. I will run it as soon as we finish with the refactoring